dir.props

```diff
@@ -131,21 +133,21 @@
 
   <!-- list of directories to perform batch restore -->
   <ItemGroup>
-    <DnuRestoreDir Include="$(MSBuildProjectDirectory)\src" />
+    <DnuRestoreDir Include="$(MSBuildProjectDirectory)/src" />
     <DnuRestoreDir Include="$(ToolsDir)" />
     <!-- workaround to address issue where DNX won't recurse directories under a project.json
          https://github.com/aspnet/dnx/commit/0dda8bf86863364cc20421f1af7494f1b2a3256f -->
-    <DnuRestoreDir Include="$(MSBuildProjectDirectory)\src\*\src\**\project.json" />
-    <DnuRestoreDir Include="$(MSBuildProjectDirectory)\src\*\ref\*\project.json" />
-    <DnuRestoreDir Include="$(MSBuildProjectDirectory)\src\*\tests\*\project.json" />
-    <DnuRestoreDir Include="$(MSBuildProjectDirectory)\src\*\*\netcore50aot\project.json" />
-    <DnuRestoreDir Include="$(MSBuildProjectDirectory)\src\Common\tests\**\project.json" />
+    <DnuRestoreDir Include="$(MSBuildProjectDirectory)/src/*/src/**/project.json" />
+    <DnuRestoreDir Include="$(MSBuildProjectDirectory)/src/*/ref/*/project.json" />
+    <DnuRestoreDir Include="$(MSBuildProjectDirectory)/src/*/tests/*/project.json" />
+    <DnuRestoreDir Include="$(MSBuildProjectDirectory)/src/*/*/netcore50aot/project.json" />
+    <DnuRestoreDir Include="$(MSBuildProjectDirectory)/src/Common/tests/**/project.json" />
   </ItemGroup>
 
   <PropertyGroup>
     <DnxPackageDir Condition="'$(DnxPackageDir)'==''">$(PackagesDir)/$(DnxPackageName)/</DnxPackageDir>
-    <DnuToolPath Condition="'$(DnuToolPath)'=='' and '$(OsEnvironment)'!='Unix'">$(DnxPackageDir)\bin\dnu.cmd</DnuToolPath>
-    <DnuToolPath Condition="'$(DnuToolPath)'=='' and '$(OsEnvironment)'=='Unix'">$(DnxPackageDir)/bin/dnu</DnuToolPath>
+    <DnuToolPath Condition="'$(DnuToolPath)'=='' and ('$(OsEnvironment)'!='Unix' or '$(OsEnvironment)'!='OSX')">$(DnxPackageDir)/bin/dnu.cmd</DnuToolPath>
```

Comment:
It is not anymore. I think that the ideal solution is not to change OsEnvironment to Unix, but instead handle the cases for OS X as well, but that is not super easy given that we have so many files that special-case Unix. Given that treating OS X as Unix was the way that builds in Mac used to work, I will remove this conditions for now, and log an issue so that we can move to special casing OSX as well.